deployment/run-unit-tests.sh

```diff
@@ -10,63 +8,26 @@
 [ "$DEBUG" == 'true' ] && set -x
 set -e
 
-prepare_jest_coverage_report() {
-  local component_name=$1
-
-  if [ ! -d "coverage" ]; then
-    echo "ValidationError: Missing required directory coverage after running unit tests"
-    exit 129
-  fi
-
-  # prepare coverage reports
-  rm -fr coverage/lcov-report
-  mkdir -p $coverage_reports_top_path/jest
-  coverage_report_path=$coverage_reports_top_path/jest/$component_name
-  rm -fr $coverage_report_path
-  mv coverage $coverage_report_path
-}
-
-run_javascript_test() {
-  local component_path=$1
-  local component_name=$2
-
+function headline(){
   echo "------------------------------------------------------------------------------"
-  echo "[Test] Run javascript unit test with coverage for $component_name"
+  echo "$1"
   echo "------------------------------------------------------------------------------"
-  echo "cd $component_path"
-  cd $component_path
-
-  # run unit tests
-  npm test
-
-  # prepare coverage reports
-  prepare_jest_coverage_report $component_name
 }
 
-# Get reference for all important folders
+headline "[Setup] Configure paths"
 template_dir="$PWD"
-source_dir="$template_dir/../source"
-coverage_reports_top_path=$source_dir/test/coverage-reports
-
-# Test the attached Lambda function
-declare -a lambda_packages=(
-  "constructs"
-  "image-handler"
-  "custom-resource"
+cdk_dir="$template_dir/../source/constructs"
+image_handler_dir="$template_dir/../source/image-handler"
+custom_resource_dir="$template_dir/../source/custom-resource"
+
+headline "[Tests] Run unit tests"
+declare -a packages=(
```

Comment:
If you have multiple packages, consider using a tool like [Lerna](https://lerna.js.org/).

Reply:
will do in subsequent releases